utils/configure-project

```diff
@@ -1,24 +1,25 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 
 set -eu
 
 fatal() {
-    printf "$*\n" > /dev/stderr
+    echo "$*" >&2
```

Comment:
printf must have a "%" pattern and arguments. Simpler to use echo.
Also this kind of redirections should be more portable (think MacOS users).

Reply:
No, printf "must" not have a "%" pattern. However the behaviour of printf and echo is quite different regarding specific characters.

Regarding /dev/stderr, I could be convinced to change it for >&2, but not in this commit. I'll think about it for all scripts.